extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -23,6 +23,7 @@ amp-sticky-ad {
   z-index: 2;
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
+  margin-bottom: 0 !important;
```

Comment:
I'm not saying we should mandate a `background`, just pointing out that `padding-bottom` can be used for the same effect with a transparent background. We need to fix the `padding-bottom` issue.